### ISSUE TITLE
"requires string as left operand, not list"

### ISSUE THREAD

**Issue by pat-s** [tfeldmann/organize#42]
OS: Arch
organize v1.4.2
`organize config --debug` -> no errors

```
organize sim               
                                                                                                                                                                              
/usr/lib/python3.7/site-packages/organize/config.py:28: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. 
```

<details>
<summary>Full Log</summary>

```
2019-08-01 15:48:38,434 - organize.main - ERROR - 'in <string>' requires string as left operand, not list
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/organize/main.py", line 102, in main
    execute_rules(config.rules, simulate=args['sim'])
  File "/usr/lib/python3.7/site-packages/organize/core.py", line 80, in execute_rules
    jobs = list(find_jobs(rules))
  File "/usr/lib/python3.7/site-packages/organize/core.py", line 47, in find_jobs
    if all(f.matches(path) for f in rule.filters):
  File "/usr/lib/python3.7/site-packages/organize/core.py", line 47, in <genexpr>
    if all(f.matches(path) for f in rule.filters):
  File "/usr/lib/python3.7/site-packages/organize/filters/filename.py", line 70, in matches
    self.contains in filename)
TypeError: 'in <string>' requires string as left operand, not list
2019-08-01 15:48:55,504 - organize.main - ERROR - 'in <string>' requires string as left operand, not list
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/organize/main.py", line 102, in main
    execute_rules(config.rules, simulate=args['sim'])
  File "/usr/lib/python3.7/site-packages/organize/core.py", line 80, in execute_rules
    jobs = list(find_jobs(rules))
  File "/usr/lib/python3.7/site-packages/organize/core.py", line 47, in find_jobs
    if all(f.matches(path) for f in rule.filters):
  File "/usr/lib/python3.7/site-packages/organize/core.py", line 47, in <genexpr>
    if all(f.matches(path) for f in rule.filters):
  File "/usr/lib/python3.7/site-packages/organize/filters/filename.py", line 70, in matches
    self.contains in filename)
TypeError: 'in <string>' requires string as left operand, not list
```

</details>

**Comment by tfeldmann** [comment]
- The YAMLLoadWarning is fixed in the new versions. Please update with `pip3 install -U organize-tool`
- The filename filter only supports simple strings for the `contains:` keyword. Support for lists will come in the next version.

**Comment by pat-s** [comment]
Thanks!

**Comment by tfeldmann** [comment]
I just published v1.5.3 which supports lists in the filename filter 👌
